From a149b89740d1a7be0a02d8de7d4beeb87ed25a3a Mon Sep 17 00:00:00 2001
From: "yi.liu" <yi.liu@amlogic.com>
Date: Mon, 10 Jul 2023 15:24:42 +0800
Subject: [PATCH 2/5] amlbian: T7b and T7c use same config [1/1]

PD#SWPL-131302

Problem:
T7b and T7c use same config.

Solution:
T7b and T7c use same config.

Verify:
local

Signed-off-by: yi.liu <yi.liu@amlogic.com>
Change-Id: I6a72e1faa9c5167d01dd0ed6977e444d7420ca96
---
 env/copy_debs_from_yocto.sh | 11 +++--------
 1 file changed, 3 insertions(+), 8 deletions(-)

diff --git a/env/copy_debs_from_yocto.sh b/env/copy_debs_from_yocto.sh
index 7be6db7..8842e08 100755
--- a/env/copy_debs_from_yocto.sh
+++ b/env/copy_debs_from_yocto.sh
@@ -138,8 +138,8 @@ get_suitable_board() {
 	elif [ ${#TARGET_MACHINE_ARRAY[*]} == 1 ]; then
 		# only one meson folder in yocto build folder : /tmp/work/
 		TARGET_BOARD=${TARGET_MACHINE_ARRAY[0]};
-	elif [ "$TARGET_MACHINE" == "mesont7c-an400-debian-5.15" ]; then
-		TARGET_BOARD=mesont7c_an400_debian_5.15-poky-linux
+	elif [ "$TARGET_MACHINE" == "mesont7-an400-debian-5.15" ]; then
+		TARGET_BOARD=mesont7_an400_debian_5.15-poky-linux
 	elif [ "$TARGET_MACHINE" == "mesong12b-w400-debian-5.15" ]; then
 		TARGET_BOARD=mesong12b_w400_debian_5.15-poky-linux
 	elif [ "$TARGET_MACHINE" == "mesonsm1-ac200-debian-5.15" ]; then
@@ -169,12 +169,7 @@ KERNEL_DEBS_DIR=$BUILDDIR/tmp/work/$TARGET_BOARD
 
 # set SOC special information and packages
 case "$TARGET_BOARD" in
-	mesont7c_an400_debian_5.15-poky-linux)	# T7C/A311D2revC
-		packages_arr+=( libmultienc gst-plugin-venc-multienc \
-			           libgles-eabihf-gondul-wayland-drm mediactrlsrc)
-		kernel_mod_arr+=( adla gondul isp)
-		;;
-	mesont7_an400_debian_5.15-poky-linux)	# T7/A311D2 B
+	mesont7_an400_debian_5.15-poky-linux)	# T7/A311D2
 		packages_arr+=( libmultienc gst-plugin-venc-multienc \
 			           libgles-eabihf-gondul-wayland-drm mediactrlsrc)
 		kernel_mod_arr+=( adla gondul isp)
-- 
2.25.1

